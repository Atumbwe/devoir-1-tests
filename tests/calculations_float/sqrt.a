200.54 sq