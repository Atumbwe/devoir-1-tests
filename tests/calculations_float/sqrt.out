 --------------------------
 <    14.161214637170076    >
 --------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||